librz/core/casm.c

```diff
@@ -99,7 +111,7 @@ RZ_API RzCmdStatus rz_core_asm_plugin_print(RzCore *core, RzAsmPlugin *ap, const
 			strcat(bits, "32 ");
 		}
 		if (ap->bits & 64) {
-			strcat(bits, "64 ");
+			strcat(bits, "64");
```

Comment:
Why this change? Other bitness values have space as well.

Reply:
Nevermind, I see that is the last change, so the space is unnecessary.

Reply:
because is always the last one and doesn't need the extra space.